### PR TITLE
chore(gatewayd): add clap --version

### PR DESCRIPTION
**Pull request by bradleystachurski**
Followup to https://github.com/fedimint/fedimint/pull/4678#discussion_r1536632435

### CODE REVIEW

**Review by m1sterc001guy — APPROVED** [review]
You beat me to it :)

**Review by dpc — APPROVED** [review]

File: gateway/ln-gateway/src/lib.rs [hunk 1]
```diff
@@ -118,6 +118,7 @@ const DEFAULT_MODULE_KINDS: [(ModuleInstanceId, &ModuleKind); 2] = [
 ];
 
 #[derive(Parser)]
+#[command(version)]
```
Comment (dpc) [inline gateway/ln-gateway/src/lib.rs:121]: While at it - worth checking if all other binaries have it.

**Comment by fedimint-backports** [comment]
Successfully created backport PR for `releases/v0.3`:
- #4706

**Review by tvolk131 — COMMENTED** [review]
Thanks for this fix Brad!

File: gateway/ln-gateway/src/lib.rs (hunk 1 above)
Comment (bradleystachurski) [reply gateway/ln-gateway/src/lib.rs:121]: Good idea 👍 

https://github.com/fedimint/fedimint/pull/4723

**Pull request merged**